web/src/components/FieldEntryAlert.tsx

```diff
@@ -13,6 +14,14 @@ interface Props {
 }
 
 export const FieldEntryAlert = (props: Props): ReactElement => {
+  const router = useRouter();
+
+  const onAnchorClick = (): void => {
+    setTimeout(() => {
+      router.push(router.asPath, undefined, { shallow: true });
+    });
+  };
```

Comment:
I'm assuming this was added to handle some issues with form data being modified or getting wiped, is that correct?

Reply:
we spoke about this